docs/general/enl.rst

```diff
@@ -0,0 +1,69 @@
+Equivalent Number of Looks (ENL)
+================================
+
+The Equivalent Number of Looks (ENL) describes the degree of averaging applied to SAR measurements during data formation
+and postprocessing and is an indirect measure of speckle reduction (e.g., due to multilooking or speckle filtering).
+
+In case of linear scaled backscatter data, ENL can be calculated as:
+
+.. math::
+    ENL = \frac{\mu^2}{\sigma^2}
+
+where :math:`\mu` is the mean and :math:`\sigma` is the standard deviation of the image. ([1], section A1.1.7)
+
+The ENL value stored in the metadata of each S1-NRB product is calculated as suggested in [2], where ENL is first
+calculated for small pixel windows over the cross-polarized backscatter image and afterwards the median value of
+the distribution is selected.
+
+Calculate ENL per image
+-----------------------
+While only the median value is currently stored in the metadata of each S1-NRB product, it is possible to calculate ENL
+as described above for entire images using the function :func:`S1_NRB.metadata.extract.calc_enl`:
+
+.. code-block:: python
+
+    from S1_NRB.metadata.extract import calc_enl
+
+    enl_arr = calc_enl(tif="s1a-iw-nrb-20220721t051225-044194-05465e-33tuf-vh-s-lin.tif", block_size=25, return_arr=True)
+
+
+Example calculated from sigma0 VH backscatter of a S1-NRB product, which has been derived from a Sentinel-1 SLC (IW) scene:
+
+.. figure:: ../_assets/enl_example_tile.png
+    :width: 85 %
+    :align: center
+    :alt: ENL calculation for an example scene
+
+
+Comparison between GRDH and NRB
```

Comment:
please add a short note in the text or image caption about what area we're looking at.